releasenotes/notes/crawler-webdriver-di-71225500f3751983.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Add an optional webdriver parameter to Crawler in order, to allow the usage of a pre-configured webdriver instead of creating a new one in the constructor
```

Comment:
```suggestion
    Add an optional webdriver parameter to Crawler to allow the usage of a pre-configured webdriver instead of creating a new one in the constructor
```